auth/permisos.ts

```diff
@@ -143,7 +143,8 @@ export default [
                 child: [
                     { key: 'medica', title: 'Médico', type: 'boolean' },
                     { key: 'enfermeria', title: 'Enfermero', type: 'boolean' },
-                    { key: 'estadistica', title: 'Estadístico', type: 'boolean' }
+                    { key: 'estadistica', title: 'Estadístico', type: 'boolean' },
+                    { key: 'carga', title: 'Carga', type: 'boolean' }
```

Comment:
```suggestion
                    { key: 'estadistica', title: 'Estadístico', type: 'boolean' },
                    { key: 'estadistica-v2', title: 'Estadístico (nuevo)', type: 'boolean' }
```

Actualizar todas las referencias en la app.

Reply:
hecho